src/utilities/fe_mesh.jl

```diff
@@ -0,0 +1,162 @@
+# Typ definitions
+type FEBoundary # Tänkte typ parametrisera detta på nåt sätt så att man har FEboundary i 2D, 1D, och 0D
+    dim::Int
+    topology::Array{Int}
+end
+
+type FEMesh{dim, T <: Real}
+    coords::Vector{Vec{dim,T}} # Node coordinates
+    topology::Matrix{Int} # Element nodes
+    boundary::Vector{Vector{}} # Boundary nodes
+end
+
+
+function FEMesh(dim)
```

Comment:
In my opinion yes. When we ask for the coordinates for an element we just compute the coordinates needed on the fly.